### ISSUE TITLE
the transition of the layout from Russian to pinyin does not work

### ISSUE THREAD

**Issue by Temurumaru** [libpinyin/ibus-libpinyin#439]
The transition of the layout from Russian to pinyin does not work.
![image](https://github.com/libpinyin/ibus-libpinyin/assets/86427523/45815bae-acc9-4480-82f8-11f8240ea467)
[Запись экрана от 2023-11-21 00-53-15.webm](https://github.com/libpinyin/ibus-libpinyin/assets/86427523/2bdeaa7d-579f-4c81-9b0d-9d2f4a433398)

**Comment by epico** [comment]
Please edit the /usr/share/ibus/component/libpinyin.xml file, change the `<layout>default</layout>` to `<layout>us</layout>`.

**Comment by Temurumaru** [comment]
Yes, not everyone has a us layout, but I think it needs to be solved somehow

**Comment by Temurumaru** [comment]
after the update, the old settings are back again. We need to do something about it. otherwise, you can't configure it with every update. after all, pinyin is already in Latin, why not leave it by default?

**Comment by Temurumaru** [comment]
终于 ：）
